includes/helper.php

```diff
@@ -705,6 +705,29 @@ public function extract_description($post_id = 0)
 		$description = $this->db->sql_fetchfield('post_text');
 		$this->db->sql_freeresult($result);
 
+		// Prefix with post subject if body length is < 25 (+ 7 for <t>...</t>), to satisfy SEO recommendations
+		if (strlen($description) < 32)
+		{
+			// Build database query for post subject
+			$sql = 'SELECT post_subject
+				FROM ' . $this->tables['posts'] . '
+				WHERE ' . $this->db->sql_build_array('SELECT', ['post_id' => $post_id]);
```

Comment:
I guess it would be possible to do a single SQL query only, `SELECT post_text,post_subject`, using the post text above and the post subject below, then freeing it afterwards. But such short post texts are very rare, and probably selecting both values marginally decreases performance (increases load) for all other post requests? The way it is now at least keeps the change a dedicated code block, easier to insert locally, test, revert etc 🙂.